src/main/java/io/fusionauth/load/FusionAuthBaseWorker.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2012-2025, FusionAuth, All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package io.fusionauth.load;
+
+import java.util.UUID;
+
+import io.fusionauth.client.FusionAuthClient;
+
+/**
+ * Worker to test creating EmailVerification Ids for users.
+ *
+ * @author Brent Halsey
+ */
+public abstract class FusionAuthBaseWorker extends BaseWorker {
+  protected final FusionAuthClient client;
+
+  private final int numberOfApplications;
+
+  private final int numberOfTenants;
+
+  protected UUID applicationId;
+
+  protected int applicationIndex;
+
+  protected UUID tenantId;
+
+  protected int tenantIndex;
+
+  protected FusionAuthClient tenantScopedClient;
+
+  public FusionAuthBaseWorker(FusionAuthClient client, Configuration configuration) {
+    super(configuration);
+    this.client = client;
+    this.numberOfApplications = configuration.getInteger("numberOfApplications", 0);
+    this.numberOfTenants = configuration.getInteger("numberOfTenants", 0);
+    this.tenantScopedClient = client;
+  }
+
+  public static UUID applicationUUID(int tenantIndex) {
```

Comment:
It was used correctly, but the variable was named wrong. Can we update all the methods in here to make the parameter name `index`? That avoids shadowing a class variable and makes it clearer where the value is coming from at a glance.
```suggestion
  public static UUID applicationUUID(int index) {
```